test/dune

```diff
@@ -1,5 +1,6 @@
 (test
   (name testsuite)
+  (enabled_if (= %{os_type} Unix))
```

Comment:
As discussed, missing tests for the port to Windows, but given the difficulty of actually doing that, I'm happy to accept this without them.